dist/api_err.py

```diff
@@ -134,19 +167,12 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
  */
 const char *
 __wt_wiredtiger_error(int error)
-{
-\t/*
-\t * Check for WiredTiger specific errors.
-\t */
-\tswitch (error) {
-''')
+{''')
```

Comment:
Sorry, I can't find where this one comes from, what is it for?

Reply:
This is for the api_strerror.c file. I made some of this part into a function, so that there is less repetition for the errors and sub-level errors. This part is the start of the `__wt_wiredtiger_error` function. It looks like this
`__wt_wiredtiger_error(int error)
{`

where the opening brace matches that new opening brace in code snippet.